hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/dispatcher/TransactionDispatcher.java

```diff
@@ -116,86 +116,81 @@ public void dispatchHandle(
-            case TOKEN_PAUSE -> handlers.tokenPauseHandler().preHandle(context);
-            case TOKEN_UNPAUSE -> handlers.tokenUnpauseHandler().preHandle(context);
+            case TOKEN_CREATION -> handlers.tokenCreateHandler();
+            case TOKEN_UPDATE -> handlers.tokenUpdateHandler();
+            case TOKEN_MINT -> handlers.tokenMintHandler();
+            case TOKEN_BURN -> handlers.tokenBurnHandler();
+            case TOKEN_DELETION -> handlers.tokenDeleteHandler();
+            case TOKEN_WIPE -> handlers.tokenAccountWipeHandler();
+            case TOKEN_FREEZE -> handlers.tokenFreezeAccountHandler();
+            case TOKEN_UNFREEZE -> handlers.tokenUnfreezeAccountHandler();
+            case TOKEN_GRANT_KYC -> handlers.tokenGrantKycToAccountHandler();
+            case TOKEN_REVOKE_KYC -> handlers.tokenRevokeKycFromAccountHandler();
+            case TOKEN_ASSOCIATE -> handlers.tokenAssociateToAccountHandler();
+            case TOKEN_DISSOCIATE -> handlers.tokenDissociateFromAccountHandler();
+            case TOKEN_FEE_SCHEDULE_UPDATE -> handlers.tokenFeeScheduleUpdateHandler();
+            case TOKEN_PAUSE -> handlers.tokenPauseHandler();
+            case TOKEN_UNPAUSE -> handlers.tokenUnpauseHandler();
 
-            case UTIL_PRNG -> handlers.utilPrngHandler().preHandle(context);
+            case UTIL_PRNG -> handlers.utilPrngHandler();
 
-            case SYSTEM_DELETE -> {
-                switch (txBody.systemDeleteOrThrow().id().kind()) {
-                    case CONTRACT_ID -> handlers.contractSystemDeleteHandler().preHandle(context);
-                    case FILE_ID -> handlers.fileSystemDeleteHandler().preHandle(context);
-                    case UNSET -> throw new IllegalArgumentException("SystemDelete without IdCase");
-                }
-            }
-            case SYSTEM_UNDELETE -> {
-                switch (txBody.systemUndeleteOrThrow().id().kind()) {
-                    case CONTRACT_ID -> handlers.contractSystemUndeleteHandler().preHandle(context);
-                    case FILE_ID -> handlers.fileSystemUndeleteHandler().preHandle(context);
-                    case UNSET -> throw new IllegalArgumentException("SystemUndelete without IdCase");
-                }
-            }
+            case SYSTEM_DELETE -> switch (txBody.systemDeleteOrThrow().id().kind()) {
+                case CONTRACT_ID -> handlers.contractSystemDeleteHandler();
+                case FILE_ID -> handlers.fileSystemDeleteHandler();
+                default -> throw new UnsupportedOperationException("SystemDelete without IdCase");
+            };
+            case SYSTEM_UNDELETE -> switch (txBody.systemUndeleteOrThrow().id().kind()) {
+                case CONTRACT_ID -> handlers.contractSystemUndeleteHandler();
+                case FILE_ID -> handlers.fileSystemUndeleteHandler();
+                default -> throw new UnsupportedOperationException("SystemUndelete without IdCase");
```

Comment:
please make it consistent if the string is inlined or a constant like for TYPE_NOT_SUPPORTED

Reply:
I am not sure I understand this comment. What needs to be made consistent?

Reply:
Sometimes you use a constant for the message of the UnsupportedOperationException and sometimes an inline string.

Reply:
Ah, understood. Will change it in the next PR.